specification.md

```diff
@@ -3503,6 +3528,15 @@ interface Breakpoint {
 }
 ```
 
+### <a name="Types_SteppingGranularity" class="anchor"></a>SteppingGranularity
+
+The granularity of the requested step.
+
+```typescript
+interface SteppingGranularity {
```

Comment:
this looks wrong. it's an empty interface ?

Where are the values defined ?

Reply:
Per other comment - digging in to this. When I re-ran the generator to remove the comments it no longer has the values.